File: iniciante/javascript/1116.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function divisao(a, b){
    if(b !== 0){
        console.log((a/b).toFixed(1))
        return 1
    }else{
        console.log('divisao impossivel')
        return 0
    }
}

let entrada = parseInt(lines.shift())
let count = 0

while(count++ < entrada){
    let entradas = lines.shift().split(' ')
    let a = parseInt(entradas[0])
    let b = parseInt(entradas[1].replace('\r',''))
    divisao(a,b)
}

File: iniciante/python/1116.py
# -*- coding: utf-8 -*-

entrada = int(input())
count_entrada = 0

while count_entrada < entrada:
    pares = input().split(' ')
    pares = [int(x) for x in pares]

    if pares[1] == 0:
        print('divisao impossivel')
    else:
        print(f'{(pares[0]/pares[1]):.1f}')

    count_entrada += 1
